src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -30,20 +30,21 @@ import { heading_to_string, radiansToDegrees, degreesToRadians } from '../utilit
  * @class AircraftCommander
  */
 export default class AircraftCommander {
-    constructor(aircraftController, onChangeTransponderCode) {
+    constructor(aircraftController) {
         this._eventBus = EventBus;
-        this._aircraftController = aircraftController;
-        this._onChangeTransponderCode = onChangeTransponderCode;
+        this._findAircraftById = aircraftController.findAircraftById.bind(aircraftController);
```

Comment:
is this binding actually needed?  if it is, could we bind these within the `aircraftController` instead?

Reply:
@n8rzz I'm not sure, I was just following @erikquinn's sugestion.

Reply:
When I suggested it, it needed the binding. Might be doable from the aircraftController, I don't know, haven't tried it.